### ISSUE TITLE
[BUG]: Action on game launch doesnt trigger

### ISSUE THREAD

**Issue by Savillion** [BartoszCichecki/LenovoLegionToolkit#806]
### Version

Latest as of now

### OS

Tried on both 10 and 11 with latest updates

### Device

Ideapad gaming 3 15arh05

### BIOS version

FCCN19WW

### What's wrong?

Opening a game with discrete gpu wont trigger the action "on game launch"

### How to reproduce the bug?

Its simple to reproduce the bug if you know and use the software. 

### What is the behavior that you expected?

It shouls trigger given events when discrete gpu is active and running a dx, vulkan or opengl based application.

### Logs

I will add later.

### Do you have Lenovo software installed?

- [ ] Lenovo Vantage is installed
- [ ] Lenovo Hotkeys is installed
- [X] None of the above

### Did you disable any Lenovo software using Lenovo Legion Toolkit?

- [ ] Lenovo Vantage
- [ ] Lenovo Hotkeys
- [X] I disabled some things on my own (please write below what you did)

### Additional information

None

**Comment by BartoszCichecki** [comment]
Did you read this: https://github.com/BartoszCichecki/LenovoLegionToolkit#why-isnt-a-game-detected-even-though-actions-are-configured-properly ?

**Comment by BartoszCichecki** [comment]
> I will add later.

I would prefer that you don't create issues that are not complete.

**Comment by Savillion** [comment]
First, sorry for this thread, I shouldn't have opened it before clean installing Windows. It works now. Even though I haven't selected "remember this is a game". It was probably a problem on my end. I tweaked a lot of things. 

I would like to add other little bugs that I encounter.
1. Single display brightness wont work when I change my power profile. Even with Windows Mobility Center or adding a custom power state and changing it via control panel. Log added.

2. My laptop doesnt have refresh rates other than 120hz, so i add 60hz with Custom Resolution Utility. Even though I can see these refresh rates within LLT, I cannot change it with fn+r. I also didn't exclude refresh rates. Maybe it's on my end, my model doesn't have 60hz on default and CRU resolutions aren't supported? Or my bios have no fn+r mapped to this function? Changing refresh rates works with custom actions though. Log added. 

3. This is a one time small issue: After installing Windows 11 updates, It doesnt run on startup, even though I selected the option and minimize to tray doesnt work. Restarting Windows fixes it.

And, thank you. I believe you are doing God's work. Providing utility tool for us for free, when Lenovo does it to sell our data.
